core/src/main/scala/io/projectglow/transformers/LiftOverVariantsTransformer.scala

```diff
@@ -250,13 +246,13 @@ object LiftOverVariantsTransformer extends GlowLogging {
     val outputVcOpt = Option(
       LiftoverUtils.liftVariant(inputVc, outputInterval, refSeq, false, false))
     if (outputVcOpt.isEmpty) {
-      return (None, Some(LiftoverVcf.FILTER_CANNOT_LIFTOVER_INDEL))
+      return (None, Some(LiftoverVcf.FILTER_CANNOT_LIFTOVER_REV_COMP))
```

Comment:
This is because of the picard update? We should make sure to document in the next release notes.

Reply:
Yep, we'll make sure to reference https://github.com/broadinstitute/picard/pull/1266 in the next release notes.